### DISCUSSION TITLE
[youtube-player]: missing 'host' in Options

### DISCUSSION THREAD

**Discussion by lgarczyn** [DefinitelyTyped/DefinitelyTyped#55655] [Issues with a @types package]
As you can see in this [discussion](https://github.com/gajus/youtube-player/issues/79), youtube-player supports the "host" parameter on player initialization.

However, the `Options` interface is missing that parameter.

I've tested said parameter without the type definition, and it works perfectly fine, but the type definition refuses it.

[a comment by an automated account is not shown]

**Comment by jurca** [comment]
@lgarczyn Hello and thank you for pointing this out! I may have overlooked this option (or the docs did not mention it at the time I made the type declarations). Would you be willing to make a PR for this?

**Reply by lgarczyn** [reply]
Hi, thanks for the answer.

I've never forked the project, and I don't have it cloned. It's a bit long just to add a single line.

**Comment by sasial-dev** [comment]
Sent a PR 🙂 (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55854)

**Reply by jurca** [reply]
Thanks, just reviewed it!
